openedx_webhooks/templates/github_community_pr_comment.md.j2

```diff
@@ -19,20 +19,24 @@ GitHub pull request interface. As a reminder,
 {% endfilter %}
 
 
-{%- if not has_signed_agreement or not in_authors_file %}
+
 {% filter replace("\n", " ")|trim %}
-We can't start reviewing your pull request until you've
   {%- if not has_signed_agreement %}
-  submitted a [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)
-  or indicated your institutional affiliation
-  {%- endif %}
-  {%- if not has_signed_agreement and not in_authors_file %}
-  and
+  We can't start reviewing your pull request until you've submitted a
+  [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)
+  or indicated your institutional affiliation.
   {%- endif %}
   {%- if not in_authors_file %}
-  added yourself to the [AUTHORS](https://github.com/{{ repo }}/blob/master/AUTHORS) file
+  This PR can only be merged when you have added yourself to the
+  [AUTHORS](https://github.com/{{ repo }}/blob/master/AUTHORS) file.
+  However, please wait for the results of the review process before
+  doing so.  When this PR is approved, then please add yourself to the
+  [AUTHORS](https://github.com/{{ repo }}/blob/master/AUTHORS) file in
+  a SEPARATE PR, and then reference it in this PR, after which BOTH
+  PRs can be merged.
```

Comment:
I think this is overly complicated, and we can simplify it a bit. How about:

"You haven't added yourself to the [AUTHORS](https://github.com/{{ repo }}/blob/master/AUTHORS) file for this repo. Please create a new pull request to do so!"

I don't think the timing is really that terribly important, and I don't want this message to seem too overwhelming. What do you think?